tests/integ/scala/test_table_suite.py

```diff
@@ -162,37 +177,46 @@ def test_multipart_identifier(session, table_name_1):
     try:
         assert session.table(name4).collect() == expected
     finally:
-        Utils.drop_table(session, name4)
+        if not local_testing_mode:
```

Comment:
I'm planning to get rid of these try-finally-drop by making all these test tables temporary tables.